apps/web-giddh/src/app/actions/login.action.ts

```diff
@@ -294,7 +294,7 @@ export class LoginActions {
                                 if (hasSubscriptionPermission) {
                                     this._router.navigate(['/pages/subscription']);
                                 } else {
-                                    this._router.navigate(['/pages/subscription/buy-plan']);
+                                    this._router.navigate(['/pages/subscription/buy-plan/' + localStorage.getItem('Country-Region')]);
```

Comment:
if there is no value in local storage, undefined may get added in url, handle it as well at both places